test/test_utils.jl

```diff
@@ -259,3 +259,15 @@ getfacerefshape(::Hexahedron, ::Int) = RefQuadrilateral
 getfacerefshape(::Tetrahedron, ::Int) = RefTriangle
 getfacerefshape(::Pyramid, face::Int) = face == 1 ? RefQuadrilateral : RefTriangle
 getfacerefshape(::Wedge, face::Int) = face ∈ (1,5) ? RefTriangle : RefQuadrilateral
+
+function perturbate_standard_grid!(grid::Ferrite.AbstractGrid{dim}, strength) where dim
+    function perturbate(x::Vec{dim}) where dim
+        for d in 1:dim
+            if x[d] ≈ 1.0 || x[d] ≈ -1.0
+                return x
+            end
+        end
+        return x + Vec{dim}(0.5*strength .* (2 .* rand(Vec{dim}) .- 1.0))
+    end
+    transform_coordinates!(grid, perturbate)
+end
```

Comment:
Just for reference, I did something similar for #798, but it was hard to make sure that negative determinants didn't occur, so decided to go with a smooth and non-random, but still nonlinear grid:
https://github.com/Ferrite-FEM/Ferrite.jl/blob/3aad963b0726571052c10053e91415bdd207aa0a/test/test_interpolations.jl#L197-L198

Reply:
Since the used perturbations in the tests are rather small negative detJ isn't an issue. I also thought about analytical deformation, but then I was not sure about which function should be applied. Also I think that the mentioned function should be extended to "rotate" some random elements just to check that we do not have hidden assumptions on the element orientation.

Reply:
Do you mean to modify "rotate" the node indices in some random cells? That would be very nice!

Reply:
Yes, cyclic shifts of the node ids, which should keep the grid intact but can trigger issues. This can be done without importing generated meshes via external packages.